chef/cookbooks/cpe_onepassword/README.md

```diff
@@ -0,0 +1,50 @@
+cpe_onepassword Cookbook
+========================
+Manage 1Password's JSON config using the CPE attributes API pattern on macOS and Windows.
```

Comment:
```suggestion
Manage 1Password 8's JSON config using the CPE attributes API pattern on macOS and Windows.
```

Reply:
I might be wrong, but I thought settings.json didn't exist on 7 and older.

Reply:
https://github.com/Gusto/it-cpe-opensource/pull/77/commits/bb9a53a6a023fac87c4600691bfffd019f35a728